news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,226 @@
+
+It's not a secret that players from South Korea are so good that they could swap each other and win the entire tournament no matter what's ahead. With no [qwhj1027](https://osu.ppy.sh/users/7547506) and [Abstract-](https://osu.ppy.sh/users/3097304) winning the previous year, some analysts had doubts that they could claim the winning title this year. Thankfully or not, that wasn't the case here at all. With 11 maps lost in this entire tournament, South Korea showed that even with not the strongest players this game could possibly meet there are plenty of players that can still impress in this mode.
+
+United States entered this tournament with almost the same roster with swapping [Kisetsunotori](https://osu.ppy.sh/users/3692648) to [Elux](https://osu.ppy.sh/users/12004983) being the only roster change. Although they've fallen to the losers bracket after playing a match against South Korea, United States showed their determination to achieve something worthy. In the end, they've claimed themselfes as grand finalists for the second time in this world cup.
+
+With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on a very top despite failing their previous run placing 8th-7th place. Their expectations worth it in the end as they instantly gained their glory reaching winners finals bracket without losing a single match until meeting with South Korea and untimately to United States. Even with a long year absense, China showed that even being absent for an entire year they can still make a good fight with top teams.
+
+There's still plenty of tournaments running though even after the conclusion of CWC! The digit tournaments, [4 Digit Catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/4CWC/2022) and [3 Digit Catch World Cup 2022](https://osu.ppy.sh/community/forums/topics/1580097 ), are up and running and a second iteration of [Catch French Dual Cup 2](https://osu.ppy.sh/community/forums/topics/1605746) currently in the registered phase. There's nothing to talk about these tournaments yet but maybe with the new osu!monthly release we can fully recap what was going on with conclusions and stuff!
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+
+#### In August, the osu!mania top players will brawl in the osu!mania World Cup 2022!
+
+As it is starting the **6th of August**, we can't actually talk about it more, but we are excited to say that the next osu!monthly is going to have a **big article** about it, and maybe an **interview of the winner**!
+
+Please stay tuned about the next article, and we're moving on the next section!
```

Comment:
Not really sure this needs to be here, not sure what the purpose is of telling the reader the next article is coming when it is pretty visible that there is more coming.

Reply:
Thought it was too short, so I wrote this
I'm deleting it